### ISSUE TITLE
output_path not working

### ISSUE THREAD

**Issue by eungangku** [zaironjacobs/get-chrome-driver#1]
@zaironjacobs 
Thank you for providing an awesome library.

This is my code. I'm trying to download Chromedriver into the same directory.
```
from get_chrome_driver import GetChromeDriver

get_driver = GetChromeDriver()
get_driver.auto_download(extract=True, output_path="")
```

and I got this
```
PS D:\Project\Python\Automation\Socinet> & C:/Python37/python.exe d:/Project/Python/Automation/Socinet/chromedriver.py
Traceback (most recent call last):
  File "d:/Project/Python/Automation/Socinet/chromedriver.py", line 4, in <module>
    get_driver.auto_download(extract=True, output_path="")
TypeError: auto_download() got an unexpected keyword argument 'output_path'
PS D:\Project\Python\Automation\Socinet> 
```

What should I do?

**Comment by zaironjacobs** [comment]
Hi, the issue is now fixed in v1.1.36. You can use output_path="" to download the driver in the same directory.
If you're using pip you can upgrade the package with "pip install get-chrome-driver --upgrade".

**Comment by zaironjacobs** [comment]
Hi @eungangku , the issue is now fixed in v1.1.36. You can use output_path="" to download the driver in the same directory.
If you're using pip you can upgrade the package with "pip install get-chrome-driver --upgrade".

**Comment by eungangku** [comment]
@zaironjacobs 
It works well. Thank you :)
